server/graphql/invoice_line/src/mutations/inbound_shipment_line/line/insert.rs

```diff
@@ -26,8 +26,8 @@ pub struct InsertInput {
     pub sell_price_per_pack: f64,
     pub expiry_date: Option<NaiveDate>,
     pub number_of_packs: u32,
-    pub total_before_tax: f64,
-    pub tax: Option<TaxUpdate>,
+    pub total_before_tax: Option<PriceInput>,
+    pub tax: Option<PriceInput>,
```

Comment:
Why not have TaxInput, and TotalBeforeTaxInput? seems a bit confusing if you just look at the api, e.g. why be able to set a percentage on total_before_tax if it is then silently ignored.